plugin/src/main/resources/plugin.yml

```
@@ -3,3 +3,4 @@ version: '0.1.0'
 main: dev.betrix.superSmashMobsBrawl.SuperSmashMobsBrawl
 api-version: '1.21'
 authors: [ BetrixDev ]
+depend: [ LibsDisguises ]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)